src/components/Navigation.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="container-fluid" style="width: 90%">
-    <nav class="row navbar navbar-expand navbar-light bg-light">
+    <nav class="row navbar navbar-expand navbar-light bg-light" v-if="loggedIn">
```

Comment:
I talked about this in the PR description: this hides the reporting period selector when you're on the login page. It doesn't make any sense there anyhow, but now it won't have any data because `updateApplicationSettings` isn't run until after login (because `/api/application_settings` now requires auth, because it needs to know what tenant to lookup settings for; on the login page we definitionally _don't know_ which tenant you'll login to yet)

Though I'm realizing now maybe you mean why hide the whole navbar vs just the period selector, and true, I could change that. Though this also pulls `applicationTitle` from `application_settings` which has the same problem mentioned above, though there is already an explicit fallback to "ARPA Reporter" -- in this case, prob would want to change this to just be a fixed title and not use that field in `application_settings` anymore?

Reply:
Oh, actually, it seems like you added an equivalent check in 4f08dcb7be3d41174a8d7c645489038f9e230fc8 after I'd done this in my branch, so it's redundant now.